<?xml version="1.0"?>
<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
        <param name="video_device" value="/dev/video0"/>
        <param name="image_width" value="320"/>
        <param name="image_height" value="240"/>
        <param name="pixel_format" value="yuyv"/>
        <param name="framerate" value="30"/>
  </node>

  <node name="detect_eyes_node" pkg="myface_recognition" type="detect_eyes_node.py" required="true" output="screen"/>
</launch>
